tools/downloader/networks_compiler.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import sys
+import common
+from pathlib import Path
+
+def compile(reporter, compiler_path, model, model_precision, args, output_dir):
+    (output_dir / model.subdirectory).mkdir(parents=True, exist_ok=True)
+
+    compile_cmd = [str(compiler_path),
+        '-m={}'.format(args.model_dir / model.subdirectory / model_precision / (model.name + '.xml')),
+        '-d={}'.format(args.target_device),
+        '-ip={}'.format('U8' if args.input_precision is None else args.input_precision),
+        '-op={}'.format('FP32' if args.output_precision is None else args.output_precision),
+        '-o={}'.format(output_dir / model.subdirectory / model_precision /(model.name + '.blob')),
+    ]
+    reporter.print_section_heading('{}Compiling {} to BLOB ({})',
+        '(DRY RUN) ' if args.dry_run else '', model.name, model_precision)
+
+    reporter.print('Conversion command: {}', common.command_string(compile_cmd))
+    if not args.dry_run:
+        reporter.print(flush=True)
+        if not reporter.job_context.subprocess(compile_cmd):
+            return False
+    reporter.print()
```

Comment:
This `print` should happen whether or not the command succeeded (otherwise, the output from a failed command will not be separated from the following section).

Reply:
Done.